spec/index.bs

```diff
@@ -513,6 +508,29 @@ interface mixin GPUObjectBase {
         An internal slot holding the [=device=] which owns the [=internal object=].
 </dl>
 
+{{GPUObjectDescriptorBase}} has the following members:
+
+<dl dfn-type=dict-member dfn-for=GPUObjectDescriptorBase>
+    : <dfn>label</dfn>
+    ::
+        The initial value of {{GPUObjectBase/label|GPUObjectBase.label}}.
+</dl>
+
+
+: <dfn dfn>Debug label</dfn>
+:: A string provided for use by development tools (such as error/warning messages,
+    browser developer tools, or platform debugging utilities) to identify the underlying
+    [=internal object=], debug group, or debug marker to the developer.
+    It has no specified format, and implementations MUST NOT use label contents in heuristics.
```

Comment:
nit: We don't normally all-caps the RFC 2119 words. Though it would be cool if they could be highlighted like in Vulkan....